pkg/github/github.go

```diff
@@ -81,6 +81,17 @@ type PullRequestResponse struct {
 	Pagination   *Pagination
 }
 
+type Job struct {
```

Comment:
Per https://google.github.io/styleguide/go/decisions#doc-comments, `All top-level exported names must have doc comments...`

Reply:
Added those for `Job` and `JobsResponse`